components/faqs/nesting.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withTheme, ThemeProvider } from 'styled-components'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+import LiveEdit from '../LiveEdit'
+
+const scope = { withTheme, ThemeProvider }
+const sample = (`
+const ColorChanger = styled.section\`
+  background: papayawhip;
+
+  > h2 {
+    color: palevioletred;
+  }
+
+  @media(min-width: 768px) {
+    background: mediumseagreen;
+
+    > h2 {
+      color: papayawhip;
+    }
+  }
+\`;
+render(
+  <ColorChanger href="#">
+    <h2>Hello world!</h2>
+  </ColorChanger>
+)
+`).trim()
+
+const Nesting = () => (
+  <SectionLayout title="Nesting">
+    <p>Nesting is a feature ported from Sass. Used sparingly it's a great way to lighten your code by reducing the need to create explicit classes for every element.</p>
```

Comment:
This slide shows nicely that you can do basically everything with nesting, that you can do with CSS (obviously). So that's very nice :)

The two interpolations are crucial, since they show that often you can combine nesting + interpolations for more powerful patterns :+1:

The only thing I'm missing is that you should introduce styled components as interpolations for their classname. But you might want to add that to an additional sub-section.